File: question7.py
# carpma islemini toplama kullanarak bulan kod

carpan1=int(input("Ilk carpani giriniz:"))
carpan2=int(input("Ikinci carpani giriniz:"))
sonuc=0
while carpan1>0:
    sonuc+=carpan2
    carpan1=carpan1-1
    
print("Carpimin sonucu:" + str(sonuc))

File: question7.java
import java.util.Scanner;
public class question7 {
    public static void main(String[] args) {
        // carpma islemini toplama kullanarak bulan kod

        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Ilk carpani giriniz: ");
        int carpan1=scanner.nextInt();

        System.out.print("Ikinci carpani giriniz: ");
        int carpan2=scanner.nextInt();

        int sonuc=0;

        while(carpan1>0){
            sonuc+=carpan2;
            carpan1--;
        }

        System.out.print("Carpimin sonucu: " + sonuc);

        scanner.close();
    }
}
